supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -41,6 +41,15 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
     JesJobExecutionActor.props(jobDescriptor, jesConfiguration, initializationData.toJes.get, serviceRegistryActor).withDispatcher(BackendDispatcher)
   }
 
+  override def cacheHitCopyingActorProps(jobDescriptor: BackendJobDescriptor,
+                                         initializationData: Option[BackendInitializationData],
+                                         serviceRegistryActor: ActorRef,
+                                         cachedJobOutputs: JobOutputs): Props = {
+    // The `get` here is safe.  If the initialization data wasn't built, the workflow execution wouldn't have advanced to the
+    // point of trying to create job cache hit copying actors.
```

Comment:
The JES initialization data should never be `None`.  If it were somehow unexpectedly `None` and we happened to make it this far into trying to create cache hit copying actors, I think I'd prefer to surface the problem rather than `map`ping in a way that disguises the failure.